### PR TITLE
[OC] Overall fix on the Performance Metrics page

### PR DESCRIPTION
**Pull request by jjasferreira**
- Fix typos and pontuation;
- Improve wording and cohesion;
- Change some text style.

### CODE REVIEW

File: content/oc/0001-metricas-performance.md [hunk 1]
```diff
@@ -293,20 +293,20 @@ $$
...
-Queremos desenhar um [processador B](color:purple) que demore
+Queremos desenhar um processador $$B$$ que demore
 [6 segundos](color:orange) de tempo de CPU a executar o mesmo programa,
-mas que necessita de [1.2 vezes mais ciclos de relógio para executar o programa](color:orange).
-**Qual será o _clock rate_ do [processador B](color:purple)?**
+mas que necessite de [1.2 vezes mais ciclos de relógio](color:yellow).
+**Qual será o _clock rate_ do processador $$B$$?**
 
-Começando por escrever a expressão do _clock rate_ de [B](color:purple), temos:
+Começando por escrever a expressão da _clock rate_ de $$B$$, temos:
```
Comment (diogotcorreia) [inline content/oc/0001-metricas-performance.md:303]: Hmm, I don't think I agree with removing the colors here, I'll revert it

**Review by diogotcorreia — APPROVED** [review]
I've reverted the colors as I said and changed inline latex to use `$` instead of `$$`, since it's more readable and also works.

File: content/oc/0001-metricas-performance.md (hunk 1 above)
Comment (jjasferreira) [reply content/oc/0001-metricas-performance.md]: Sure, that's way better! I though you couldn't combine latex with colors.

**Comment by jjasferreira** [comment]
Sure, that's way better! I though you couldn't combine latex with colors.

**Comment by diogotcorreia** [comment]
@jjasferreira You can, and there's even something better.
If you want to change the color of just a few characters inside the LaTeX block, you can use a special macro I created:
```latex
something \smartcolor{green}{this will be in green} something
```

This uses the website colors, so it follows light/dark mode preferences (and the hidden theme engine that I still have to build the UI for)

https://github.com/leic-pt/resumos-leic/blob/a043284de75e5e3fef7703c26af3497deb458e76/gatsby-config.js#L90

**Comment by jjasferreira** [comment]
@diogotcorreia Thanks for the info!

**Pull request merged**